cpp/util/utils.h

```diff
@@ -47,34 +48,59 @@ inline void forLoopWrapper(size_t begin, size_t end, const Body& body, bool para
 }
 
 //! Wrapper for 2D nested for loops to allow the execution in parallel or not.
-/*! \param parallel If true, run body in parallel.
- *  \param begin_row Beginning index of outer loop.
+/*! \param begin_row Beginning index of outer loop.
  *  \param end_row Ending index of outer loop.
  *  \param begin_col Beginning index of inner loop.
  *  \param end_col Ending index of inner loop.
  *  \param body An object with operator(size_t begin_row, size_t end_row, size_t begin_col, size_t end_col).
+ *  \param parallel If true, run body in parallel.
  */
 template<typename Body>
-inline void forLoopWrapper2D(
-    size_t begin_row, size_t end_row, size_t begin_col, size_t end_col,
-    const Body& body, bool parallel = true)
+inline void forLoopWrapper2D(size_t begin_row, size_t end_row, size_t begin_col, size_t end_col,
+                             const Body& body, bool parallel = true)
 {
     if (parallel)
     {
         tbb::parallel_for(tbb::blocked_range2d<size_t>(begin_row, end_row, begin_col, end_col),
-                [&body](const tbb::blocked_range2d<size_t>& r) {
-                    body(r.rows().begin(), r.rows().end(), r.cols().begin(), r.cols().end());
-                }
-        );
+                          [&body](const tbb::blocked_range2d<size_t>& r) {
+                              body(r.rows().begin(), r.rows().end(), r.cols().begin(), r.cols().end());
+                          });
     }
     else
     {
         body(begin_row, end_row, begin_col, end_col);
     }
 }
 
-};
+//! Wrapper for 3D nested for loops to allow the execution in parallel or not.
+/*! \param begin_page Beginning index of outer loop.
+ *  \param end_page Ending index of outer loop.
+ *  \param begin_row Beginning index of middle loop.
+ *  \param end_row Ending index of middle loop.
+ *  \param begin_col Beginning index of inner loop.
+ *  \param end_col Ending index of inner loop.
+ *  \param body An object with operator(size_t begin_page, size_t end_page, size_t begin_row, size_t end_row, size_t begin_col, size_t end_col).
+ *  \param parallel If true, run body in parallel.
+ */
+template<typename Body>
+inline void forLoopWrapper3D(size_t begin_page, size_t end_page, size_t begin_row, size_t end_row,
+                             size_t begin_col, size_t end_col, const Body& body, bool parallel = true)
+{
+    if (parallel)
+    {
+        tbb::parallel_for(
+            tbb::blocked_range3d<size_t>(begin_page, end_page, begin_row, end_row, begin_col, end_col),
+            [&body](const tbb::blocked_range3d<size_t>& r) {
+                body(r.pages().begin(), r.pages().end(), r.rows().begin(), r.rows().end(), r.cols().begin(),
+                     r.cols().end());
+            });
+    }
+    else
+    {
+        body(begin_page, end_page, begin_row, end_row, begin_col, end_col);
+    }
+}
```

Comment:
Let's remove this function. We can add it at a later date if we have a need for it, but we have enough experience with how long cruft lies around in freud that I don't want to carry around unused functions without a demonstrated need. Also remove the corresponding blocked_range3d include at the top.

Reply:
I don't think this is within scope of the PR, I agree that the change should be made, but let's do this somewhere else.

Reply:
@tommy-waltmann I'm not sure I follow your logic. This function is being added by this PR, so choosing _not_ to add it definitely seems in scope for this PR.